node/tests/TestUtilities.ts

```diff
@@ -634,7 +645,7 @@ export async function transactionTest(
         baseTransaction.smismember(key7, ["bar", "foo", "baz"]);
         responseData.push([
             'smismember(key7, ["bar", "foo", "baz"])',
-            [true, true, false],
```

Comment:
why?

Reply:
There was a mistake. I detected it once I reworked result checking to use deep equality check.